### ISSUE TITLE
Easy way to change default adapter

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#105]
**Reported by jcsiadal on 20 Aug 2003 02:22 UTC**
```
Request: Option to change the default interface on the 
nodes, currently eth0.

This comes from many installations on some of our 
systems where the out-of-band hardware management 
interface is enumerated as eth1.

I'm also going to be testing TCP over IB and iANS, 
which require different interfaces.

Right now, this requires me to edit the systeminstaller 
master script file immediately after creating the client 
image.

How about adding another text field at the Define Oscar 
Clients dialog labeled Default Interface?
```

**Comment by dikim33** [comment]
**Comment by brechin on 24 Feb 2004 19:08 UTC**
```

This is not a terribly difficult task, but it should be 
implemented with the new GUI.
```

**Comment by dikim33** [comment]
**Comment by brechin on 13 Jul 2004 16:50 UTC**
```

Not maintaining this code any longer, so I'm unassigning 
myself.
```

**Comment by dikim33** [comment]
**Comment by valleegr on 7 Dec 2008 06:36 UTC**
(In [7677]) This check-in fixes #106 - It is now possible to change the headnode NIC using the ''oscar-config --headnode-nic command''.
